content/en/docs/contributing/third-party-code-donation.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Donating Third Party Code to cert-manager"
+linkTitle: "Third Party Code Donations"
+weight: 120
+type: "docs"
+---
+
+The cert-manager project welcomes external contributions and has benefited greatly from thousands
+of commits from hundreds of different contributors. Most code is usually committed through pull
+requests to a specific repo, whether that be the main cert-manager repository or one of the associated
+repositories such as the website.
+
+Some contributions aren't as well suited to that kind of workflow, however. That would most likely
+be because their functionality doesn't belong in any particular existing cert-manager repo, while still
+relating to the cert-manager project.
+
+This document aims to address the donation of code to the cert-manager project, and to provide a
+framework for sustainable contributions which can be tested and relied upon going forwards by both
+cert-manager maintainers and users.
+
+The requirements in this document are based in part on what's done for CoreDNS, Envoy, Kubernetes
+and containerd.
+
+## Requirements
+
+- Code must be licensed appropriately, including any dependencies   
```

Comment:
worth using numbers? Easier for referencing to specific points.

Reply:
If you use `### name`, you can link to anchors...

But, yeah, bullets are probably least recommended for things like this. Some identifier would be preferable.

Reply:
Agreed that an ordered list beats an unordered list here, I'll make that change.